projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+* ✅ Suitable when all the events created through your application all have the same location
+* ✅ Suitable when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+* ❌ Not suitable when the locations of the events created through your application are unpredictable in advance
+
+#### Example
+
+In the example below, all the events created through the integration take place in "Het Depot", a concert hall in Leuven. First you'll need to find the identifier of the place `Het Depot` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank Search API, including the name of the place and the postal code:
+
+```
+GET /places/?q=name.\*:"Het Depot" AND address.\*.postalCode:3000
+```
+
+**response**
+
+The response now contains the identifier of "Het Depot":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7",
+         "@type":"Place"
+      }
+   ]
+}
+```
+
+You can now store the identifier of the place "Het Depot" (`8248e289-c986-4006-902f-b0616dcbcde7`) in your application to reuse it as location ID in the `POST` request of the event(s) that you want to create.
+
+### Searching existing locations within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the locations for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same location
+* ❌ Not recommended when the events created through your application can have a different location, but (1) the amount of possible locations is limited & (2) the list of possible locations is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Het Depot" in the input field for the location. Under the hood an API call to the UiTdatabank Search API is made to check if that place already exists:
+
+**request**
+
+```
+GET /places/?q=name.\*:"Het Depot" AND address.\*.postalCode:3000
+```
+
+**response**
+
+The response now contains the identifier of "Het Depot":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/place/8248e289-c986-4006-902f-b0616dcbcde7",
+         "@type":"Place"
+      }
+   ]
+}
+```
+
+If the place already exists (like in the example above), it is suggested in the user interface in a human-readable way so that the user can select the place and thus choose to reuse the place for the creation of his or her event.
```

Comment:
```suggestion
If one or more places already exist with this name in the given postal code (like in the example above), they are displayed as human-readable suggestions in the user interface so that the user can choose to reuse one of these place for the creation of his or her event.
```

(The "it is suggested" reads as something that we suggest to the integrator at first glance)